server/odc-service/src/main/java/com/oceanbase/odc/service/feature/VersionDiffConfigService.java

```diff
@@ -135,6 +140,23 @@ public List<OBSupport> getSupportFeatures(ConnectionSession connectionSession) {
         return obSupportList;
     }
 
+    public boolean isExternalTableSupported(@NonNull DialectType dialectType, @NonNull Connection conn) {
```

Comment:
involve unnecessary metadb query for each 'listTables' API, may:

1. VersionDiffConfigService should not depends on Connection and Plugin, it is a configuraion module which is designed for depended by other modules
2. externalTableSupported may initialized while ConnectionSession created
3. VersionDiffConfigService  may involve cache for better performance